src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1719,8 +1719,8 @@
   <data name="DetectPreviewFeaturesDescription" xml:space="preserve">
     <value>An assembly has to opt into preview features before using them.</value>
   </data>
-  <data name="DetectPreviewFeaturesMessage" xml:space="preserve">
-    <value>Using '{0}' requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+  <data name="DetectPreviewFeaturesMessageWithCustomMessagePlaceholder" xml:space="preserve">
+    <value>{2} Using '{0}' requires opting into preview features. See {1} for more information.</value>
```

Comment:
The url here added without quote but for other messages enclosed in a quite, want to make it same for all, which message looks better?
`Using 'Foo' requires opting into preview features. See https://aka.ms/aspnet/kestrel/http3reqs for more information.` or 
`Using 'Foo' requires opting into preview features. See 'https://aka.ms/aspnet/kestrel/http3reqs' for more information.`